package.json

```diff
@@ -27,7 +27,8 @@
     "refapp3SearchAndRegistration": "cypress run --spec src/test/resources/features/refapp-3.x/02-search-and-registration/search-and-registration.feature",
     "refapp3Settings": "cypress run --spec src/test/resources/features/refapp-3.x/03-settings/settings.feature",
     "refapp3ClinicalVisit": "cypress run --spec src/test/resources/features/refapp-3.x/04-clinical-visit/clinical-visit.feature",
-    "refapp3VitalsAndTriage": "cypress run --spec src/test/resources/features/refapp-3.x/05-vitals-and-triage/vitals-and-triage.feature"
+    "refapp3VitalsAndTriage": "cypress run --spec src/test/resources/features/refapp-3.x/05-vitals-and-triage/vitals-and-triage.feature",
+    "refappOCLSubscriptionModule": "cypress run --spec src/test/resources/features/oclsubscriptionmodule/index.feature"
```

Comment:
Same here, we can make it like **refappOclSubscriptionModule.**

Reply:
Oh I see the convention, thanks I will update that.